lib/polyfills/set.js

```diff
@@ -0,0 +1,15 @@
+if (!Set.prototype.union) {
```

Comment:
of these 3 methods, only `union` is actually used, and just once.
tbh I would avoid the hassle to define and have to require the polyfills set here and there and just inline the `union` method where it's used (worker)

Reply:
For me this felt odd to not have difference and intersections, but I agree. I removed the other ones. 
Unfortunately when I move this `union` definition into the worker file, then it is not possible to add a unit test for this (see: https://github.com/tidalcycles/pulsar-tidalcycles/pull/223#discussion_r2265266563)